tests/python/pants_test/option/test_options_integration.py

```diff
@@ -233,6 +233,38 @@ def test_invalid_command_line_option_and_invalid_config(self):
       self.assertIn("ERROR] Invalid option 'bad_option' under [test.junit]", pants_run.stderr_data)
       self.assertIn("ERROR] Invalid scope [invalid_scope]", pants_run.stderr_data)
 
+  def test_invalid_command_line_option_shows_suggestions_in_error(self):
```

Comment:
Could you add a test for the suffix-in-wrong-scope piece too?

Reply:
Done!